spanner_orm/admin/api.py

```diff
@@ -56,9 +56,14 @@ def connect(cls,
   @classmethod
   def _connection(cls):
     if not cls._spanner_connection:
-      raise error.SpannerError('Not connected to spanner')
+      raise error.SpannerError('Not connected to Spanner')
     return cls._spanner_connection
 
+  @classmethod
+  def drop_database(cls):
+    cls._connection.drop()
```

Comment:
This needs to be on _spanner_connection

Reply:
Dang, I thought I had remembered to add the parens here. I was trying to call the method, not use the variable